Source/RestKit/RestRequest.swift

```diff
@@ -303,7 +303,36 @@ public struct RestRequest {
             completionHandler(dataResponse)
         }
     }
-    
+
+    public func responseVoid(
```

Comment:
Any other names for this function that come to mind? For some reason I find `responseVoid` confusing. It makes sense in the context of the other functions (e.g. `responseObject`, `responseArray`, etc.), but doesn't seem to stand on its own.

If I were to stumble across this function a few months from now, I might think that it somehow "voids" the request/response by short-circuiting something about the HTTP request or response from the service.

That being said, I can't think of a great alternative. The two best that come to mind are `responseEmpty` and `responseNothing`. But I'm not sure if they're any better than `responseVoid`.

What do you think?